src/main/java/io/openliberty/tools/common/plugins/util/DevUtil.java

```diff
@@ -1006,20 +1020,49 @@ public void run() {
         }
     }
 
-    private String runCmd(String cmd) throws IOException, InterruptedException {
-        return runCmd(true, cmd);
+    private Process getRunProcess(String command) throws IOException {
+        ProcessBuilder processBuilder = new ProcessBuilder();
+        processBuilder.command(getCommandTokens(command));
+        return processBuilder.start();
     }
 
-    private String runCmd(boolean report, String cmd) throws IOException, InterruptedException {
+    private void runCommandAndLog(final Process startingProcess, int timeout) throws InterruptedException {
```

Comment:
Should this be renamed to something like `runDockerCommandAndLog`?  Although it's not related to Docker except for its debug message string.

Perhaps rename this method and/or `execDockerCmd` method to avoid confusion, since both method names sound like they do the same thing.

Reply:
How about execDockerCmd() and execDockerCmdAndLog()? They do the same thing except the one that logs?

Reply:
That's fine.